site/samples/work_manager.md

```diff
@@ -0,0 +1,63 @@
+## Work Manager Sample
+
+This WDT domain model sample section has typical configurations for a Work Manager and its related Request Classes and Constraints. These elements are configured in the `SelfTuning` folder in the `resources` section of the model.
```

Comment:
Request Classes and Constraints -> request classes and constraints (no need to capitalize these terms)